qa-core/src/account-api/api/AccountInternalAPIClient.ts

```diff
@@ -39,6 +43,13 @@ export default class AccountInternalAPIClient {
         credentials: "include",
       }
 
+      if (options.internal) {
+        requestOptions.headers = {
+          ...requestOptions.headers,
+          ...{ "x-budibase-api-key": env.ACCOUNT_PORTAL_API_KEY },
```

Comment:
shouldn't need to spread here - just add the `"x-budibase-api-key":  env.ACCOUNT_PORTAL_API_KEY` directly

Reply:
I had it already like that but it complained because of there is no type definition and the original var doesn't have this one